docs/user/tutorial/index.rst

```diff
@@ -287,6 +287,11 @@ When you are satisfied, you can merge the pull request!
 Customizing the build process
 -----------------------------
 
+.. warning::
+
+   We are updating this tutorial.
+   :doc:`As of September 2023, you need to add a configuration file as mentioned in these steps <rtd-blog:migrate-configuration-v2>`.
+
```

Comment:
This seems like an odd warning. I think we should include information for the user in the tutorial, instead of this warning. This would just make me stop doing the tutorial if I saw it.

Reply:
I agree that this is not optimal. But it's inside a section about adding a configuration file, that's why I found it to be "better than nothing".

Reply:
I mean, the warning says "as mentioned in these steps" and the sections contains some good instructions about adding that file.

It can certainly be improved, but this is just short-term improvement.

Reply:
Could you clarify what you mean by the comment? Do you want me to turn the warning into more clear instructions? This risks jump-starting changes to the tutorial.

Reply:
We can rename the section to "Adding a config file" ?

Reply:
@ericholscher I added a bit more guidance but am afraid that this will escalate :)

Reply:
I still think this text is weird, and will scare the users...
```suggestion
.. warning::
   As of September 2023,
   :doc:`a configuration file will be required  to build documentation<rtd-blog:migrate-configuration-v2>`. 
```

Reply:
I think it looks good now. I also clarified that following the tutorial still works. I think it's nice to be explicit about that and not expect someone following the tutorial to jump out and grab more information from the blog post.